simpleclient_jetty_jdk8/pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If you mean maven compiler source and target, I did in the subsequent lines. I don't actually understand what you are referring to?